### ISSUE TITLE
Jest fails with errors that say Cannot find module 'babel-types' or Cannot find module 'babel-template'

### ISSUE THREAD

**Issue by jbseppanen** [heap/react-native-heap#424]
Index file found here requires `babel-types` and `babel-template` packages.  https://github.com/heap/react-native-heap/blob/develop/instrumentor/src/index.js#L3-L4

I attempted to replace those two imports with this:

`const t = require('@babel/types');`
`const template = require('@babel/template');`

That then changes the error to say `TypeError: [BABEL]: template is not a function`.

This came from the PR [here](https://github.com/heap/react-native-heap/pull/421).  I wanted that version (0.22.7) for the security audit fix that it brings.  I'm not clear on what need to happen with the template in order to fix.

**Comment by bomjkolyadun** [comment]
Seeing the same issue with 0.22.7

**Comment by bnickel** [comment]
Is this issue just limited to Jest?  Is autocapture still working correctly?

**Comment by bnickel** [comment]
Okay, I'm seeing the issue as well.  I've determined the fix is:

```
const t = require('@babel/types');
const template = require('@babel/template').default;
```

I'll push a fix later today.  Thanks for reporting this.

**Comment by bnickel** [comment]
The fix has been released as 0.22.8
